lib/fog/aws/models/storage/file.rb

```diff
@@ -315,22 +325,15 @@ def multipart_copy(options, upload_part_options, target_directory_key, target_fi
 
           # Store ETags of upload parts
           part_tags = []
+          pending = create_part_list(upload_part_options)
 
-          current_pos = 0
+          thread_count = options['concurrency'] || 10
```

Comment:
I wonder if it would be safer to default to 1 to start, and then once we have vetted it a bit we could bump it up to a higher number? I was thinking this might simplify debugging/etc if anything comes up, while allowing people to opt-in to the other behavior. Not set on one way or the other, but wanted to check-in and see your feeling.

Reply:
Done. 

I made `concurrency` an attribute rather than a parameter `options` because I worried that the option would be passed along the HTTP request by this or other Fog providers. What do you think?